app/utils/clearHolders.ts

```diff
@@ -0,0 +1,15 @@
+import { Address } from "ton";
+import { Engine } from "../engine/Engine";
+import { holdersUrl } from "../engine/corp/ZenPayProduct";
+import { extractDomain } from "../engine/utils/extractDomain";
+
+export function clearHolders(engine: Engine, address?: Address) {
```

Comment:
This function would be more clear if the `address` won't be optional

Reply:
👍